### PR TITLE
PAQ: Use inline images again, and some other changes

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 1]
```diff
@@ -48,83 +48,59 @@ If the navigator is currently hidden, click the small arrow at the very left of
...
 
 #### Step 7: Save the model and go back to the model manager {#step-7-save-the-model-and-go-back-to-the-model-manager}
 
-1.  In the toolbar of the model editor, click the following icon to save your newly created model:
-    ![Save icon](/images/streaming-analytics/analytics-builder/icon-save.png)
-
-2.  In the toolbar of the model editor, click the following icon to leave the model editor and thus to return to the model manager:
-    ![Close icon](/images/streaming-analytics/analytics-builder/icon-exit.png)
+1. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-save.png" alt="Save icon" style="display:inline-block; margin:0"> to save your newly created model.
+2. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-exit.png" alt="Close icon" style="display:inline-block; margin:0"> to leave the model editor and thus to return to the model manager
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/getting-started.md:103]: > click the close icon

File: content/streaming-analytics/analytics-builder-bundle/getting-started.md [hunk 2]
```diff
@@ -181,33 +151,23 @@ The following is a brief description of the blocks that are defined within the s
...
-
-5.  In the toolbar of the model editor, click the following icon to leave the model editor and thus to return to the model manager:
-    ![Save icon](/images/streaming-analytics/analytics-builder/icon-exit.png)
-
-    {{< c8y-admon-info>}}
+1. Go to the **Samples** tab of the model manager.
+2. Click the actions menu of the **On missing measurements create alarm** sample and then click **Create model from sample**.
+   The new model is immediately shown in the model editor. It has the same name, description and tags as the sample.
+3. If you want to rename the model, click the model name which is shown at the left of the toolbar. You can then specify a new name in the resulting **Model Configuration** dialog box.
+4. In the toolbar of the model editor, click <img src="/images/streaming-analytics/analytics-builder/icon-save.png" alt="Save icon" style="display:inline-block; margin:0"> to save the new model.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/getting-started.md:158]: See above

File: content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md [hunk 3]
```diff
@@ -28,8 +28,7 @@ Each row that is shown for an instance has an actions menu \(the three vertical
 
 You can control the list of instances by filtering and sorting. See [Filtering and sorting the instances](/streaming-analytics/analytics-builder/#filtering-and-sorting-the-instances) for more information.
 
-If the following error icon is shown near the end of a row, the corresponding instance is no longer processing events. Click that icon to get more information.
-![Error icon](/images/streaming-analytics/analytics-builder/instance-editor-error.png)
+If the <img src="/images/streaming-analytics/analytics-builder/instance-editor-error.png" alt="Error icon" style="display:inline-block; margin:0"> error icon is shown near the end of a row, the corresponding instance is no longer processing events. Click that icon to get more information.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:31]: Please have the icon name (here "error icon") being followed by the image

File: content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md [hunk 4]
```diff
@@ -73,15 +71,12 @@ Once an instance is active, you cannot modify the template parameter values or m
...
-
     You cannot activate instances that are in draft mode.
 
 2.  If you have selected simulation mode, click the calendar icon which is now shown, specify the time span that is to be used, and click **Apply**. See also [Simulation parameters](/streaming-analytics/analytics-builder/#simulation-parameters).
 
 3.  When the toggle button in the **Status** column currently shows **Inactive**, click this button to change the state to **Active**. For simulation mode, you can only set the state to **Active** when a valid time range has been defined.
-
-    In the case of an error, the following error icon is shown at the right of the table and the instance cannot be activated. Click the error icon to get more information.
-    ![Error icon](/images/streaming-analytics/analytics-builder/instance-editor-error.png)
+    In the case of an error, the <img src="/images/streaming-analytics/analytics-builder/instance-editor-error.png" alt="Error icon" style="display:inline-block; margin:0"> error icon is shown at the right of the table and the instance cannot be activated. Click the error icon to get more information.
```
Comment (BeateRixen) [inline content/streaming-analytics/analytics-builder-bundle/using-the-instance-editor.md:79]: See above

**Comment by BeateRixen** [comment]
@kailol-sag @MWindel I started a review and added comments but we should first align on some general rules.

1. Icon names are not capitalized or formatted in a special way, so it is "save icon, close icon,". At least that is the way we use it right now.
2. Inline images for icons follow the icon name (plus "icon"), i.e. "save icon + image"
3. We should not use an icon without naming it, otherwise the reading flow gets interrupted. Exception could perhaps be the "X icon" as this simply shows an "X".

I have added several comments but stopped my review then to first align with you.

What do you think?

**Comment by kailol-sag** [comment]
> @kailol-sag @MWindel I started a review and added comments but we should first align on some general rules.
> 
> 1. Icon names are not capitalized or formatted in a special way, so it is "save icon, close icon,". At least that is the way we use it right now.
> 2. Inline images for icons follow the icon name, i.e. "save icon + image"
> 3. We should not use an icon without naming it, otherwise the reading flow gets interrupted. Exception could perhaps be the "X icon" as this simply shows an "X".
> 
> I have added several comments but stopped my review then to first align with you.
> 
> What do you think?

Ok, @BeateRixen , I can change the wording for the inline icons as you suggested. 

But I'm confused now: I looked into several other places of the doc and only found icon names in the running text, but no image. Is this a new convention that I'm not aware of? However, I find the images helpful and would therefore prefer to keep them.

File: content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md [hunk 5]
```diff
@@ -119,22 +99,18 @@ You can also create a new model from a sample. See [Creating a model from a samp
...
 3.  Click **OK**.
-
     The model editor appears. See [Overview of steps for adding a model](/streaming-analytics/analytics-builder/#overview-of-steps-for-adding-a-model) for a brief overview of how to add blocks and wires to the new model.
 
 ### Editing an existing model {#editing-an-existing-model}
 
 You can edit \(or view\) each model that is currently listed in the model manager.
 
-When a model is active, editing will set the model to read-only mode. In this case, the model editor only allows you to view the contents of the model \(for example, you can view the block parameters\). You can navigate and zoom the model as usual, but you cannot change anything. The save button in the model editor is therefore disabled.
+When a model is active, editing will set the model to read-only mode. In this case, the model editor only allows you to view the contents of the model \(for example, you can view the block parameters\). You can navigate and zoom the model as usual, but you cannot change anything. The save icon in the model editor is therefore disabled.
```
Comment (MWindel) [inline content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:113]: @kailol-sag I would show the icon here. What do you think?

Comment (kailol-sag) [reply content/streaming-analytics/analytics-builder-bundle/using-the-model-manager.md:113]: ok, I'll insert it.

**Review by MWindel — APPROVED** [review]

**Pull request merged**